source/logs/command-history.txt

```diff
@@ -32,3 +32,37 @@ text editor:
 
    * - Windows
      - ``%UserProfile%/.mongodb/mongosh/mongosh_repl_history``
+
+Starting in |mdb-shell| 2.4.0, you can use the ``history()`` command to
+view a list of previously executed commands. For example:
```

Comment:
We should state somewhere that the history is returned in chronological order (as opposed to reverse chronological order, which is also a popular choice in other tools)

Reply:
Done